{"uuid":"3da4d6a9-0288-4e8c-bb45-71695ccea416","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["ee7b09b2-e2bd-4560-8e27-3844f137ecab"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382245543,"stop":1673382245543}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382245573,"stop":1673382245573},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382246498,"stop":1673382246498},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382246831,"stop":1673382246831},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"3b7098fd-0367-4fa1-a5d5-428ae79892e8-attachment","type":"image/png"},{"name":"Myscreenshot","source":"540fc388-765d-41a3-b463-3f2d6ee7ec43-attachment"}],"parameters":[],"start":1673382246855,"stop":1673382247286}],"start":1673382245543,"stop":1673382247286}